Frends.AzureDataLake.DeleteDirectory/Frends.AzureDataLake.DeleteDirectory/Definitions/Input.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace Frends.AzureDataLake.DeleteDirectory.Definitions;
+
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+/// <summary>
+/// Input class usually contains parameters that are required.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Which connection method should be used for connecting to Azure Data Lake.
+    /// </summary>
+    /// <example>ConnectionMethod.ConnectionString</example>
+    [DefaultValue(ConnectionMethod.ConnectionString)]
+    public ConnectionMethod ConnectionMethod { get; set; }
+
+    /// <summary>
+    /// Connection string to Azure Data Lake.
+    /// </summary>
+    /// <example>DefaultEndpointsProtocol=https;AccountName=accountname;AccountKey=Pdlrxyz==;EndpointSuffix=core.windows.net</example>
+    [DisplayFormat(DataFormatString = "Text")]
+    [PasswordPropertyText]
+    [UIHint(nameof(ConnectionMethod), "", ConnectionMethod.ConnectionString)]
+    public string ConnectionString { get; set; }
+
+    /// <summary>
+    /// Name of the Azure Data Lake account.
+    /// </summary>
+    /// <example>Storager</example>
```

Comment:
lets make this lower case, because storage accs in Azure cannot have upper letter at all